index.md

```diff
@@ -157,23 +166,21 @@ Budapest felől érkezve, csak Pest megyei matricával, elég gyorsan megközel
+Az egyik legjobb magyar szerpentin. Nagyon jó vonalvezetés, vannak szűk és kevésbé szűk kanyarok, jó a ritmusuk. Sok az emelkedő, lejtő és döntött kanyar is, így nagy a dinamikája. Még sikán is akad pár. Jó útminőség Cered irányából, közepes útminőség Zagyvaróna irányából. Valamivel keskenyebb, mint a 24-es. Alacsony a forgalom az úton, jó eséllyel lehet zavartalanul haladni. Rónabányához közel van pár nagyon éles, szűkülő visszafordító ahol igencsak le kell lassítani, erre fontos koncentrálni.
 
 <iframe src="https://www.google.com/maps/embed?pb=!1m24!1m12!1m3!1d3487.6403431963345!2d19.877696856291614!3d48.11955682276526!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!4m9!3e0!4m3!3m2!1d48.142944299999996!2d19.960096099999998!4m3!3m2!1d48.116059799999995!2d19.864380399999998!5e0!3m2!1shu!2shu!4v1593378242234!5m2!1shu!2shu" width="100%" height="450" frameborder="0" style="border:0;" allowfullscreen="" aria-hidden="false" tabindex="0" loading="lazy"></iframe>
 
-### Hangony - Zabar (utolsó frissítés: 2021 június)
+### Hangony - Zabar (utolsó frissítés: 2021 szeptember)
 7 pont
 
-Keletről nyugatra haladva ez az útvonal egy igazán hangulatos szakasz. Hangony után egy enyhén kanyargó, tempós útvonallal kezdődik, mely körül változatos és szép kilátásban gyönyörködhetünk. A Megyehatárhoz közeledve hegyessé válik a környezet és egy szűk, izgalmas szerpentin dobja fel az adrenalint. Amennyiben nyugat felé haladunk, nagyon vigyázzunk az [utolsó, lejtmeneti hajtűkanyarnál](https://goo.gl/maps/nQ2WiaupZrbcq7Rh6), ugyanis itt könnyen el lehet szállni nagy tempónál. Ez az útvonal egy tökéletes bevezető a Ceredi szerpentinhez. Az útminőség nagyon jó.
-
-<iframe src="https://www.google.com/maps/embed?pb=!1m24!1m12!1m3!1d42565.14252409505!2d20.101087828672902!3d48.18115852244082!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!4m9!3e0!4m3!3m2!1d48.2235623!2d20.191308499999998!4m3!3m2!1d48.142228499999995!2d20.0533947!5e0!3m2!1shu!2shu!4v1597068491624!5m2!1shu!2shu" width="100%" height="450" frameborder="0" style="border:0;" allowfullscreen="" aria-hidden="false" tabindex="0" loading="lazy"></iframe>
+Keletről nyugatra haladva ez az útvonal egy igazán hangulatos szakasz. Hangony után egy enyhén kanyargó, tempós útvonallal kezdődik, mely körül változatos és szép kilátásban gyönyörködhetünk. A Megyehatárhoz közeledve hegyessé válik a környezet és egy szűk, rövid, izgalmas szerpentin dobja fel az adrenalint. Amennyiben nyugat felé haladunk, nagyon vigyázzunk az [utolsó, lejtmeneti hajtűkanyarnál](https://goo.gl/maps/nQ2WiaupZrbcq7Rh6), ugyanis itt könnyen el lehet szállni nagy tempónál. Ez az útvonal egy tökéletes bevezető a Ceredi szerpentinhez. Az útminőség nagyon jó.
 
-### Balaton - Borsodnádasd
+Eger felől a Balaton településen keresztül érdemes megközelíteni.
```

Comment:
(igazából a Szentdomonkos-Borsodnádasd szakasz is élvezetes, csak azon többször voltunk már)